src/type/schema.js

```diff
@@ -250,6 +281,13 @@ export class GraphQLSchema {
     return this._possibleTypeMap[abstractType.name][possibleType.name] != null;
   }
 
+  isImplementation(
+    interfaceType: GraphQLInterfaceType,
+    possibleType: GraphQLObjectType | GraphQLInterfaceType,
+  ): boolean {
+    return this.getImplementations(interfaceType).includes(possibleType);
```

Comment:
Includes is not supported in IE11 so we can't use it until we drop support for it :(

Reply:
Also please check other places for functions not supported in IE11

Reply:
I didn't realize that IE11 was a target for this library. I removed all instances of `Array.prototype.includes`, including one that wasn't introduced in this PR. I know that with TS you can specify the library available in the target... this is something we should make sure to set correctly during the move to TS to catch any of these cases.

Reply:
I'll keep my new changes from `.includes` to `.indexOf` for now, but reading through the babel configs, it looks like `.includes` should be fine, because it is rewritten with a polyfill when targeting IE11.

Reply:
Err... it looks like the other language features that would require an IE11-targeting app to include `babel-polyfill` are only used in the tests, so my claim here doesn't hold true; I reverted my change to the code not introduced by this PR, since it was in a test.